src/components/Tally.tsx

```diff
@@ -12,7 +13,15 @@ export default class Tally extends React.Component<IProps> {
       ticks.push(null);
     }
     return (
-      <span style={{ fontFamily: "monospace" }}>{ticks.map(() => "|")}</span>
+      <span style={{ fontFamily: "monospace" }}>
+        {ticks.map(() => (
+          <FaBug
+            key={Math.random()
+              .toString(36)
+              .substr(2, 5)}
```

Comment:
A random key seems like something that will just make the tools happy without any real benefit. In this case, since the elements are interchangeble, I would make the key the index from the map. Something like this, maybe?

```jsx
{ticks.map((_, index) => <FaBug key={index} />)}
```

Reply:
I talked to @Osmose last night and he liked the bugs.